### PR TITLE
Add Roman number to integer solution

### PR DESCRIPTION
**Pull request by sank8dalvi**
Question link:
https://www.interviewbit.com/problems/roman-to-integer/

### CODE REVIEW

File: Hashing/RomanNumberToInteger [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+            return 0
+        dict = {"I": 1, "V": 5, "X": 10, "L": 50, "C": 100, "D": 500, "M": 1000}
+        sum = 0
+        firstChar = dict[A[-1]]
+        sum += firstChar
+        for i, char in enumerate(A[::-1]):
+            if i == 0:
+                continue
+            if dict[char] < dict[A[-i]]:
+                sum -= dict[char]
```
Comment (amitrajitbose) [inline Hashing/RomanNumberToInteger]: Better to not name variables similar to reserved keywords. Some IDEs might throw warnings.

Comment (sank8dalvi) [reply Hashing/RomanNumberToInteger]: Thanks for the suggestion. I have made the required changes.

File: Hashing/RomanNumberToInteger [hunk 2]
```diff
@@ -0,0 +1,20 @@
+class Solution:
+    # @param A : string
+    # @return an integer
+    def romanToInt(self, A):
+        if A=="":
+            return 0
+        dictionary = {"I": 1, "V": 5, "X": 10, "L": 50, "C": 100, "D": 500, "M": 1000}
+        totalSum = 0
+        firstChar = dictonary[A[-1]]
```
Comment (amitrajitbose) [inline Hashing/RomanNumberToInteger]: @sank8dalvi 
There's a typo. Please take your time and verify it properly. :)

**Review by amitrajitbose — CHANGES_REQUESTED** [review]
There is one typo that would lead to an error. Please fix.

File: Hashing/RomanNumberToInteger (hunk 2 above)
Comment (sank8dalvi) [reply Hashing/RomanNumberToInteger]: Very sorry for this issue and I have fixed it.

**Comment by sank8dalvi** [comment]
I have also tested my latest code on Interviewbit and it solves all the test cases.

**Comment by amitrajitbose** [comment]
Thanks, @sank8dalvi 
Your contribution is much appreciated.

**Comment by sank8dalvi** [comment]
Thank you for fixing my issues and merging!

**Pull request merged**